EapWebInterface/src/main/java/de/hpi/unicorn/application/pages/input/generator/DependenciesPanel.java

```diff
@@ -226,6 +235,30 @@ public void onSubmit(final AjaxRequestTarget target, final Form form) {
     }
 
     /**
+     * Create and add a link to delete the selected attribute dependency.
+     */
+    private void addDeleteDependencyButton() {
+        AjaxLink deleteDependencyButton = new AjaxLink<Void>("deleteDependencyButton") {
+            private static final long serialVersionUID = 1L;
+
+            @Override
+            public void onClick(final AjaxRequestTarget target) {
+                AttributeDependency dependency = AttributeDependency
+                        .getAttributeDependencyIfExists(selectedEventType, selectedBaseAttribute, selectedDependentAttribute);
+                if (dependency == null) {
+                    DependenciesPanel.this.page.getFeedbackPanel().error("Error while deleting dependency. Dependency is already deleted.");
+                } else if (dependency.remove() == null) {
+                    DependenciesPanel.this.page.getFeedbackPanel().error("Error while deleting dependency. Please Delete the corresponding values first.");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Split this 155 characters long line (which is greater than 150 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=squid%3AS00103)